1-exercises/A-accessing-values/exercise3.js

```diff
@@ -21,6 +21,9 @@ let basketballTeam = {
 */
 
 // write code here
+let sortTopPlayers = basketballTeam.topPlayers.sort();
+
+console.log(sortTopPlayers);
```

Comment:
this is good, but can you think of a way to log the name of each player on a new line?